package.json

```diff
@@ -2,20 +2,40 @@
   "name": "@cheqd/mcp",
   "version": "1.0.0",
   "description": "Package with all the mcp toolkits for cheqd",
-  "workspaces": [
-    "packages/*"
-  ],
   "private": "true",
-  "scripts": {
-    "format": "prettier --write '**/*.{js,ts,cjs,mjs,json}'"
+  "packageManager": "pnpm@9.1.3",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cheqd/mcp-toolkit.git"
   },
+  "scripts": {
+    "verify": "lerna run verify --stream",
+    "build": "lerna run build",
+    "test": "lerna run test",
+    "release": "multi-semantic-release",
+    "lint": "lerna run --parallel --no-bail lint",
+    "format": "find packages -maxdepth 1 -type d -exec sh -c 'cd {} && [ -f package.json ] && pnpm format' \\;",
```

Comment:
Fails because there's no definition of `format` in the packages:

```bash
ankur@Heart-of-Gold ~/w/i/mcp-toolkit (monorepo)> pnpm run format

> @cheqd/mcp@1.0.0 format /Users/ankur/workspace/identity/mcp-toolkit
> find packages -maxdepth 1 -type d -exec sh -c 'cd {} && [ -f package.json ] && pnpm format' \;

 ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command "format" not found
 ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command "format" not found
 ```